akka-cluster/src/main/scala/akka/cluster/protobuf/ClusterMessageSerializer.scala

```diff
@@ -228,7 +226,9 @@ final class ClusterMessageSerializer(val system: ExtendedActorSystem)
   private def poolToProto(pool: Pool): cm.Pool = {
     val builder = cm.Pool.newBuilder()
     val serializer = serialization.findSerializerFor(pool)
-    builder.setSerializerId(serializer.identifier).setData(ByteString.copyFrom(serializer.toBinary(pool)))
+    builder
+      .setSerializerId(serializer.identifier)
+      .setData(ByteStringUtils.toProtoByteStringUnsafe(serializer.toBinary(pool)))
```

Comment:
Could you also add a warning comment in the Serializer api docs that the array returned from toBinary must be read-only (not changed afterwards).